internal/ingest/linksystem.go

```diff
@@ -631,3 +674,19 @@ func isHAMT(n ipld.Node) bool {
 	h, _ := n.LookupByString("hamt")
 	return h != nil
 }
+
+func stringsToMultiaddrs(addrs []string) []multiaddr.Multiaddr {
```

Comment:
This can probably be moved to `storetheindex/mautil` if you happen to rebase before you merge.

Reply:
Noted, will do